### PR TITLE
UCP/GTEST: Fix setting mask for available tls.

### PR DESCRIPTION
**Pull request by alinask**

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/6998/ for details (Mellanox internal link).

File: src/ucp/core/ucp_context.c [hunk 1]
```diff
@@ -99,6 +99,10 @@ static ucs_config_field_t ucp_config_table[] = {
    "establishing client/server connection. ",
    ucs_offsetof(ucp_config_t, sockaddr_aux_tls), UCS_CONFIG_TYPE_STRING_ARRAY},
 
+  {"WARN_INVALID_CONFIG", "y",
+   "Issue a warning in case of non-available device and/or transport.",
```
Comment (yosefe) [inline src/ucp/core/ucp_context.c]: "Issue a warning in case of invalid device and/or transport configuration"

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7011/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7012/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7019/ for details (Mellanox internal link).

**Pull request merged**